go.mod

```diff
@@ -1 +1,3 @@
 module github.com/lib/pq
+
+go 1.13
```

Comment:
this should be reverted

Reply:
running go test keeps bringing it back, urgh